src/runtime/gecko/localization.js

```diff
@@ -0,0 +1,57 @@
+import { Localization, contexts, translateDocument }
+  from '../../bindings/html';
+import { valueFromContext } from '../../lib/format';
+
+// Services.jsm is imported in entry points: l20n-html and l20n-xul
```

Comment:
Why not load it here? rollup gets confused about undefined variable `Services` and can reassign it's name...